Instacard/models/SelectedImageModel.swift

```diff
@@ -27,24 +27,19 @@ class SelectedImageModel: ObservableObject {
     
     struct SelectedImage: Transferable {
         let image: Image
+        let cgImage: CGImage
```

Comment:
I added this extra property as a convenience. UIImage has a cgImage property which can be used to access the CGImage. However, we convert UIImage to Image and there is no way to go back to UIImage from Image. So, I kept a reference to cgImage for convenience as we'll need it for working with the vision framework.